tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -90,13 +101,23 @@ def test_ceph_mgr_dashboard_not_deployed():
 
 
 @tier1
+@metrics_for_external_mode_required
 @pytest.mark.bugzilla("1779336")
 @pytest.mark.polarion_id("OCS-1267")
 def test_ceph_rbd_metrics_available():
     """
     Ceph RBD metrics should be provided via OCP Prometheus as well.
     See also: https://ceph.com/rbd/new-in-nautilus-rbd-performance-monitoring/
     """
+    if (
```

Comment:
Do we need this check if we are already using `metrics_for_external_mode_required` mark at L#104

Reply:
I missed out on removing those lines. Removed them now